docker/lnt.llvm.org/main.tf

```diff
@@ -0,0 +1,48 @@
+#
+# Terraform file for deploying lnt.llvm.org.
+#
+
+provider "aws" {
+  region = "us-west-2"
+}
+
```

Comment:
We also need a way to set the terraform state. We use a GCS bucket in the premerge cluster to do this. https://github.com/llvm/llvm-zorg/blob/87d07e600970abf419046d2ab6083b2d64240bce/premerge/main.tf#L31

Otherwise state isn't saved across checkouts, which means things won't work.